test/jdk/java/nio/charset/Charset/Contains.java

```diff
@@ -103,6 +105,24 @@ static void checkUTF() throws Exception {
                    true);
     }
 
+    static void selfContainmentTest() {
```

Comment:
Be nice to add a comment regarding the purpose of the test

Reply:
The spec is that Charset::contains return true for self so maybe it could be renamed to containsSelfTest, might help with the understanding the purpose (along with a comment)

Reply:
Good point. Will address these in the next iteration